client/packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx

```diff
@@ -20,8 +20,14 @@ const StyledCurrencyInput = styled(RCInput)(({ theme }) => ({
   padding: '4px 8px',
   textAlign: 'right',
   border: 'none',
-  '&:focus': {
+  '&:focus:not(hover)': {
     outline: 'none',
+    borderBottom: '2px solid #3e7bfa',
```

Comment:
Is it possible to use the 'secondary' color here instead of the colour value?

Reply:
[a3a56df](https://github.com/openmsupply/open-msupply/pull/882/commits/a3a56df569ee12f671034a2e0b0c5eed34fd2bd3)